pages/common/csvstat.md

```diff
@@ -0,0 +1,28 @@
+# csvstat
```

Comment:
I just removed one of the examples (the sums and means commands work exactly the same, so I removed the latter)